encord/user_client.py

```diff
@@ -775,6 +777,14 @@ def create_storage_folder(
     def get_storage_folder(self, folder_uuid: UUID) -> StorageFolder:
         return StorageFolder._get_folder(self._api_client, folder_uuid)
 
+    def get_client_metadata_schema(self, organisation_id: int) -> Optional[Dict[str, ClientMetadataSchemaTypes]]:
+        return get_client_metadata_schema(self._api_client, organisation_id)
+
+    def set_client_metadata_schema_from_dict(
+        self, organisation_id: int, json_dict: Dict[str, ClientMetadataSchemaTypes]
+    ) -> None:
+        return set_client_metadata_schema_from_dict(self._api_client, organisation_id, json_dict)
```

Comment:
This return maybe misleading, as it gives an impression that the method returns something, while it really isn't.
Matter of taste though

Reply:
Removed the return